packages/components-react/button-react/package.json

```diff
@@ -19,31 +19,19 @@
     "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
-    "@babel/plugin-transform-runtime": "7.24.7",
-    "@babel/preset-env": "7.24.7",
-    "@babel/preset-react": "7.24.7",
-    "@babel/preset-typescript": "7.24.7",
-    "@rollup/plugin-babel": "6.0.4",
-    "@rollup/plugin-commonjs": "26.0.1",
-    "@rollup/plugin-json": "6.1.0",
-    "@rollup/plugin-node-resolve": "15.2.3",
-    "@rollup/plugin-typescript": "12.1.0",
     "@testing-library/dom": "8.20.1",
     "@testing-library/jest-dom": "6.5.0",
     "@testing-library/react": "16.0.1",
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/button-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```

Comment:
I believe the rollup can also be removed.

Reply:
We can't remove it, because `rollup` is still in `pnpm run build`